codespell_lib/data/dictionary.txt

```diff
@@ -7641,11 +7641,11 @@ deamonise->daemonise, daemonize,
 deamonised->daemonised, daemonized,
 deamonises->daemonises, daemonizes,
 deamonising->daemonising, daemonizing,
-deamonization->daemonisation, daemonization,
-deamonize->daemonise, daemonize,
-deamonized->daemonised, daemonized,
-deamonizes->daemonises, daemonizes,
-deamonizing->daemonising, daemonizing,
+deamonization->daemonization, daemonisation,
+deamonize->daemonize, daemonise,
```

Comment:
Should these also suggest `demonise` etc.?

Reply:
I would want to put this in an different merge request. because one can do a lot with d(a)emon* in the dictionary. 
I started to have a look on some of them [here](https://github.com/sebweb3r/codespell/tree/daemon)